http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala

```diff
@@ -221,18 +223,18 @@ class LeonardoService(protected val dataprocConfig: DataprocConfig,
   def processClusterCreationRequest(userInfo: UserInfo,
                                     googleProject: GoogleProject,
                                     clusterName: ClusterName,
-                                    clusterRequest: ClusterRequest): Future[Cluster] = {
-    val traceId = TraceId(randomUUID)
+                                    clusterRequest: ClusterRequest)(implicit ev: ApplicativeAsk[IO, TraceId]): Future[Cluster] = {
     for {
-      _ <- checkProjectPermission(userInfo, CreateClusters, googleProject)
-
+      _ <- checkProjectPermission(userInfo, CreateClusters, googleProject).unsafeToFuture()
+      traceId <- ev.ask.unsafeToFuture()
+_ = logger.info(s"[$traceId] successfully checked project permission")
```

Comment:
Indentation.. or maybe this statement should be removed?

Reply:
should be removed :)